Core/Utilities/ProcessPool.py

```diff
@@ -870,14 +872,14 @@ def processResults(self):
     :param self: self reference
     """
     processed = 0
-    log = gLogger.getSubLogger('ProcessPool')
+    log = LOG.getSubLogger('ProcessPool')
```

Comment:
This name would now be `DIRAC.Core.Utilities.ProcessPool.ProcessPool`, would it make more sense to use `WorkingProcess` as the last level?, or just `log = LOG`

Reply:
Agree